### DISCUSSION TITLE
target practice mod causes issues with skins made in stable. (even the default one)

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22269] [Q&A] [answered]
When using the target Practice mod on very high bpm song (sample is 777bpm) ALL hitcircles on display flash in this weird way.

I demoed it using 2013 osu default skin and some others which look the exact same you can believe me on that part. 

# seizure warning

https://user-images.githubusercontent.com/87947227/213044574-f8e1af65-329e-4853-8c5a-9366053eb649.mp4

lazerskins however dont seem to do this or atleast in a very toned down manner where I cant spot it.

https://user-images.githubusercontent.com/87947227/213045038-2a7e0c2c-a8e3-4027-add3-9ab5135cae0e.mp4

https://user-images.githubusercontent.com/87947227/213045050-c73117e8-b5ce-4b1d-9f66-4b8d2dc1db1b.mp4

So this is probably a backwards compatibility thing that should be dealt with soon or just put a seizure warning there per default in the meantime to avoid "those" kinds of issues. (health ones primarily)

**Comment by peppy** [comment, marked as answer]
That just looks like kiai time. It's being toned down in the next release.

Also, no, that cannot cause a seizure.

**Reply by Inconsist** [reply]
Since I didnt know I wanted to make sure. Thanks for the info.
